test/test_utils.jl

```diff
@@ -6,6 +6,7 @@ Random.seed!(1234)
 fder(x, y) = exp(y) * x + y * log(x)
 dfderdx(x, y) = exp(y) + y * 1 / x
 dfderdy(x, y) = exp(y) * x + log(x)
+dfderdxdx(x, y) = -y * 1 / x^2
```

Comment:
Is there a reason to prefer this over the IMO more natural
```suggestion
dfderdxdx(x, y) = -y / x^2
```
?

Reply:
No reason; I was just following the style from above